src/Controller/API/AcademicYears.php

```diff
@@ -69,10 +73,11 @@ public function getAll(
         Request $request,
         CourseRepository $courseRepository,
         SerializerInterface $serializer,
-        ApiResponseBuilder $builder
+        ApiResponseBuilder $builder,
+        AcademicYearFactory $academicYearFactory
     ): Response {
-        $years = array_map(function ($year) {
-            return new AcademicYear($year);
+        $years = array_map(function ($year) use ($academicYearFactory) {
```

Comment:
Wanna make this our first use of the https://www.php.net/manual/en/functions.arrow.php syntax?

Reply:
nice. done. :heavy_check_mark:

Reply:
Sweet!!!